### PR TITLE
Use Https instead of Http in InstallNode scripts

### PR DESCRIPTION
**Pull request by wtgodbe**

### CODE REVIEW

**Review by adityamandaleeka — APPROVED** [review]

**Comment by adityamandaleeka** [comment]
> [One too many S's](https://github.com/dotnet/aspnetcore/pull/49764/commits/fdcf0898f88a8bc2ac188872763b28a2b0f6dba9)

Can never be too secure...

**Review by sebastienros — APPROVED** [review]

File: eng/helix/content/InstallNode.ps1 [hunk 1]
```diff
@@ -1,13 +1,13 @@
- <# 
- .SYNOPSIS 
-     Installs NodeJs from http://nodejs.org/dist on a machine
- .DESCRIPTION 
-     This script installs NodeJs from http://nodejs.org/dist on a machine. 
+ <#
+ .SYNOPSIS
+     Installs Node.Js from https://nodejs.org/dist on a machine
```
Comment (adityamandaleeka) [inline eng/helix/content/InstallNode.ps1:3]:
```suggestion
     Installs Node.js from https://nodejs.org/dist on a machine
```

File: eng/helix/content/InstallNode.ps1 [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
+ .DESCRIPTION
+     This script installs Node.Js from https://nodejs.org/dist on a machine.
```
Comment (adityamandaleeka) [inline eng/helix/content/InstallNode.ps1:5]:
```suggestion
     This script installs Node.js from https://nodejs.org/dist on a machine.
```

File: eng/helix/content/InstallNode.ps1 [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
  .PARAMETER Version
-     The version of NodeJS to install.
- .LINK 
+     The version of Node.JS to install.
```
Comment (adityamandaleeka) [inline eng/helix/content/InstallNode.ps1:7]:
```suggestion
     The version of Node.js to install.
```

File: eng/helix/content/InstallNode.ps1 [hunk 4]
```diff
@@ -32,8 +32,8 @@ if (Test-Path "$InstallDir\node.exe")
 }
 
 $nodeFile="node-v$Version-win-x64"
-$url="http://nodejs.org/dist/v$Version/$nodeFile.zip"
-Write-Host "Starting download of NodeJs ${Version} from $url"
+$url="https://nodejs.org/dist/v$Version/$nodeFile.zip"
+Write-Host "Starting download of Node.Js ${Version} from $url"
```
Comment (adityamandaleeka) [inline eng/helix/content/InstallNode.ps1:36]:
```suggestion
Write-Host "Starting download of Node.js ${Version} from $url"
```

File: eng/helix/content/InstallNode.ps1 [hunk 5]
```diff
@@ -53,7 +53,7 @@ else {
     [System.IO.Compression.ZipFile]::ExtractToDirectory("nodejs.zip", $tempDir)
 }
 
-Write-Host "Expanded NodeJs to $tempDir, moving $tempDir\$nodeFile to $InstallDir subdir"
+Write-Host "Expanded Node.Js to $tempDir, moving $tempDir\$nodeFile to $InstallDir subdir"
```
Comment (adityamandaleeka) [inline eng/helix/content/InstallNode.ps1:56]:
```suggestion
Write-Host "Expanded Node.js to $tempDir, moving $tempDir\$nodeFile to $InstallDir subdir"
```

**Review by adityamandaleeka — APPROVED** [review]

**Pull request merged**